packages/ember-metal/lib/weak_set.js

```diff
@@ -0,0 +1,23 @@
+const HAS_WEAK_SET = typeof WeakSet === 'function';
+
+class WeakSetPolyFill {
+  construcor() {
+    this._weakmap = new WeakMap();
+  }
+
+  add(val) {
+    this._weakmap.set(val, true);
+    return this;
+  }
+
+  delete(val) {
+    this._weakmap.delete(val);
+    return this;
```

Comment:
This should be `return this._weakmap.delete(val);`. It's a boolean that tells you whether an element was actually deleted or not.

Reply:
ok thanks